packages/perseus/src/widgets/interactive-graphs/locked-figures/locked-function.tsx

```diff
@@ -34,7 +42,11 @@ const LockedFunction = (props: LockedFunctionType) => {
     }
 
     return (
-        <g className="locked-function">
+        <g
+            className="locked-function"
+            aria-label={hasAria ? props.ariaLabel : undefined}
+            aria-hidden={!hasAria}
```

Comment:
Why is `aria-hidden` needed here? Does the screenreader read something for this SVG element if there's no `aria-label`?

Reply:
Yes! It says "empty group"